CONTRIBUTING.md

```diff
@@ -47,22 +47,25 @@ there is a bug and the faster we'll be able to take action.
 
 ## Requesting a Feature
 
-1. Search Issues for similar feature requests. It's possible somebody has
```

Comment:
one adjustment, would be:

1. RFC PR for well formed ideas
2. RFC issues for proposed rough ideas – basically a great place to test the waters.